omod/src/main/compass/sass/login.scss

```diff
@@ -8,4 +8,9 @@
       width: 405px;
     }
   }
-}
\ No newline at end of file
+}
+
+.location-error {
+	color: red;
```

Comment:
i didn't know there was an existing error class. I have looked at it, it would probably work fine. I am testing it now. I have not seen the css file where the styles for field-error are declared though. I have seen just the referenceapplication.css, but this is generated at runtime and git is set to ignore these. Can you please point me to the .scss file where field error is declared, in case i need to add some more styles? :)